github-actions/google-internal-tests/action.yml

```diff
@@ -7,6 +7,10 @@ inputs:
   github-token:
     required: true
     description: GitHub Token used for creating the commit statuses.
+  sync-branch:
```

Comment:
IMHO I don't think we should have this as a setting, it should just always be `main`.  Particularly because if we allow another branch to be used, then we need to leverage our branch targeting logic from ng-dev.  `main` is the one exception as it must be targeted via the github UI to be included.

Reply:
Updated.